Index: back/reserve/src/main/java/sejong/reserve/domain/Member.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/back/reserve/src/main/java/sejong/reserve/domain/Member.java b/back/reserve/src/main/java/sejong/reserve/domain/Member.java
--- a/back/reserve/src/main/java/sejong/reserve/domain/Member.java	
+++ b/back/reserve/src/main/java/sejong/reserve/domain/Member.java	
@@ -55,6 +55,7 @@
         }
         this.cnt = cnt;
     }
+
     @OneToOne(mappedBy = "member", cascade = CascadeType.REMOVE)
     private Reservation reservation_log;
 }
